### DISCUSSION TITLE
How to increase latitude?

### DISCUSSION THREAD

**Discussion by imariasantos** [Stellarium/stellarium#1749] [Q&A] [answered]
Based on the API documentation [to get the location](http://stellarium.org/doc/head/classStelMainScriptAPI.html#ab68c8e4409696f19bc36d3f384d2b0a9) and [set the location](http://stellarium.org/doc/head/classStelMainScriptAPI.html#aa6655a5dc00083085c3fc12646536231), I tried to do the following.

```
// Increase the latitude by 10 degrees.
location = core.getObserverLocationInfo();
core.setObserverLocation(location.longitude, location["latitude"]+10, location.altitude);
```

When I run this, nothing seems to happen. How can I increase the latitude of the observer in a script?

**Comment by gzotti** [comment]
I can confirm it does not work as expected in 0.21.0. Does apparently nothing.

**Reply by alex-w** [reply]
I'll check it tonight - looks like a bug

**Reply by alex-w** [reply]
Yes, this is a bug and he was introduced by me, when I fixed other bug :(

**Comment by alex-w** [comment, marked as answer]
The bug is fixed in revision d33a1a89240a5189a30eb4829b6d734341dec765.

Workaround for version 0.21.0:
```
// Increase the latitude by 10 degrees.
location = core.getObserverLocationInfo();
core.setObserverLocation(location["longitude"], location["latitude"]+10, location["altitude"], 0.5, "", location["planet"]);
```
